docs/docs-new/pages/product/workspace/semantic-layer-sync.mdx

```diff
@@ -265,6 +265,45 @@ sending them, or use a service account with a different email address:
 
 <Screenshot src="https://ucarecdn.com/2e7ac466-73a5-4dd6-9c85-ed98d9df8979/" />
 
+### Tableau
```

Comment:
Since we now have four tool-specific subsections on the page, should we extract them into subpages to prevent this page from being, um, too bloated?

<img width="225" alt="Screenshot 2023-07-25 at 13 44 19" src="https://github.com/cube-js/cube/assets/3852894/e4ff1c10-5b82-4eac-94fa-c701071fbbfd">

Reply:
Agreed, but that work should be part of a separate task/PR